query-node/schemas/proposals.graphql

```diff
@@ -351,6 +351,11 @@ type UpdateGlobalNftLimitProposalDetails @variant {
   newWeeklyNftLimit: Int
 }
 
+type SetEraPayoutDampingFactorProposalDetails @variant {
+  "Proposed validator payout damping factor"
+  dampingFactor: Int!
```

Comment:
Isn't `Float` a better type of damping factor?
```suggestion
  dampingFactor: Float!
```

Reply:
I thought the Percent type was an integer (and for more precision Permill, Perbill, ... where used). Can the Percent value be a decimal ? Or is there a different reason why it should be a Float ?